allocate.go

```diff
@@ -1,9 +1,11 @@
 package ipfscluster
 
 import (
+	"context"
 	"errors"
 	"fmt"
 
+	"github.com/gxed/opencensus-go/trace"
```

Comment:
The import path and the license issues have been fixed. The duplicate dep issue is between third party dependencies, so I am unsure how to resolve this. Some packages are using version 3.6.0 of the protobuf package, others are using the 3.6.1 version.

Reply:
```
│  ├─ opencensus-go                        QmUfweMsFkBY84W8s462fkcfkUt9fygHVi8fBfwMqoCZZ3 0.0.5
│  │  ├─ client_golang                     QmYBuBECFyX3FCVNYTV1T1ACTj34qn1BwDHKJJiL1xsA3Q 0.1.4
│  │  │  ├─ protobuf                       QmZHU2gx42NPTYXzw6pJkuX6xCE7bKECp6e8QcPdoLx8sx 3.6.0
│  │  │  ├─ common                         QmTWEDbLX2d4NiMgPks6J2crRz47BamBtP16WiFuTL6Ydm 0.0.0
│  │  │  │  ├─ protobuf                    QmZHU2gx42NPTYXzw6pJkuX6xCE7bKECp6e8QcPdoLx8sx 3.6.0
│  │  │  │  ├─ golang_protobuf_extensions  QmQzJ5r1Ku82GvPwMEPtFhq3S3ZtC5LnEsS8zhTMPBPYw3 0.0.0
│  │  │  │  │  └─ protobuf                 QmZHU2gx42NPTYXzw6pJkuX6xCE7bKECp6e8QcPdoLx8sx 3.6.0
│  │  │  │  └─ client_model                QmYaVovLzgcdBpCLEAnW41p8ujvCUxe3TFpfJxjK5qbzn7 0.0.0
│  │  │  │     └─ protobuf                 QmZHU2gx42NPTYXzw6pJkuX6xCE7bKECp6e8QcPdoLx8sx 3.6.0
│  │  │  └─ client_model                   QmYbmhizinPw6KAs89nM2pWG6eD7gsGeANjYXEeZkZtVeJ 0.0.1
│  │  │     └─ protobuf                    QmTNEsrsAFXXdBxjZjdoN35QxFdeSdsdGUJBFVwdRX6ySk 3.6.1
│  │  ├─ protobuf                          QmTNEsrsAFXXdBxjZjdoN35QxFdeSdsdGUJBFVwdRX6ySk 3.6.1
│  │  └─ grpc-go                           QmSkr1sXi5iAkKi4nYusstn5npVnZXhHxiP921Q9fCptGi 0.0.2
│  │     ├─ go-genproto                    QmZRxomdAqLHT4unKFDEFEzvq4Dp9VFCvHF6qR28TDqNyh 0.0.2
│  │     │  └─ protobuf                    QmTNEsrsAFXXdBxjZjdoN35QxFdeSdsdGUJBFVwdRX6ySk 3.6.1
│  │     └─ protobuf                       QmTNEsrsAFXXdBxjZjdoN35QxFdeSdsdGUJBFVwdRX6ySk 3.6.1
```

`client_golang` uses sone and `grpc-go` uses other.  These should be aligned and bubbled to `opencensus-go`. Also please squash and force-push the gx release commits in those gxed repos when fixing so that the git history is clean and there's a single gx release per version number.

Reply:
I believe I have fixed this finally.

Reply:
@hsanjuan No I haven't fixed this apparently. `protobuf` is imported by `badger` :sob: 
```
├─ go-libp2p                               QmdJdFQc5U3RAKgJQGmWR7SSM7TLuER5FWz5Wq6Tzs2CnS 6.0.29
│  ├─ go-libp2p-secio                      QmQoDgEDBPJ6RfKDgdoMpPwoK4WEZHUosHoUs6YVDGoen3 2.0.24
│  │  └─ go-conn-security                  QmVovmja8iXHy1JouPULFdKExUrGutwzgptJZEAFG9rL1t 0.1.22
│  │     └─ go-libp2p-net                  QmNgLg1NTw37iWbYPKcyK85YJ9Whs1MkPtJwhfqbNYAyKg 3.0.22
│  │        └─ go-libp2p-peerstore         QmPiemjiKBC9VA7vZF82m4x1oygtg2c2YVqag8PX7dN1BD 2.0.13
│  │           └─ go-ds-badger             QmVoK2ivqzp5ZgWiEdBNFbKH7nzf9C4wPYr8cH7CGPMHtC 1.11.1
│  │              └─ badger                QmdG6rxQFetuV853nsNQn9EbrUEjQcbPB2htDniuLTQBsg 2.10.0
│  │                 └─ protobuf           QmZHU2gx42NPTYXzw6pJkuX6xCE7bKECp6e8QcPdoLx8sx 3.6.0
```